fusionX/Dtos/FeedbackDtoUpdate.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace hackweek_backend.dtos
+{
+    public class FeedbackDtoUpdate
+    {
+        public string Response { get; set; }
```

Comment:
Está com "warning" nesta linha, adicionar valor padrão pra removê-lo:
```suggestion
        public string Response { get; set; } = string.Empty;
```